sim/rogue/rupture.go

```diff
@@ -86,13 +89,13 @@ func (rogue *Rogue) registerRupture() {
 }
 
 func (rogue *Rogue) RuptureDamage(comboPoints int32) float64 {
-	return 127 +
-		18*float64(comboPoints) +
+	return 142 +
+		20*float64(comboPoints) +
 		[]float64{0, 0.06 / 4, 0.12 / 5, 0.18 / 6, 0.24 / 7, 0.30 / 8}[comboPoints]*rogue.Rupture.MeleeAttackPower()
 }
 
 func (rogue *Rogue) RuptureTicks(comboPoints int32) int32 {
-	return 3 + comboPoints + core.TernaryInt32(rogue.HasMajorGlyph(proto.RogueMajorGlyph_GlyphOfRupture), 2, 0)
+	return 3 + comboPoints + core.TernaryInt32(rogue.HasPrimeGlyph(proto.RoguePrimeGlyph_GlyphOfRupture), 2, 0)
```

Comment:
cache the ticks from the glyp and dont use the hasGlyph methods outside of register

Reply:
It actually already did this - missed removing this as VSCode doesn't mark Public functions as unused I think.

Removing both RuptureDuration and RuptureTicks as they are unused.